src/components/activity-page/activity-page-content.tsx

```diff
@@ -45,7 +49,14 @@ export class ActivityPageContent extends React.PureComponent <IProps> {
         {page.is_hidden && this.renderHiddenWarningBanner()}
         {this.renderPageChangeNotification()}
         <div className={`page-content full ${isResponsiveLayout ? "responsive" : ""}`} data-cy="page-content">
-          <div className="name">{ pageTitle }</div>
+          <div className="header">
+            <div className="name">{ pageTitle }</div>
+            {this.props.setReadAloud && <ReadAloudToggle
+              disabled={this.props.readAloudDisabled}
+              isChecked={this.props.readAloud || false}
+              onChange={this.props.setReadAloud}
+            />}
```

Comment:
These few lines are also copied and pasted into multiple comments. Maybe some helper function that could take props could be useful?

It feels to me that MobX or another library that lets us avoid passing all the properties from top to bottom could simplify AP quite a bit. But that's obviously another task.

Reply:
Seems like React context would make sense in this context, since I believe it's a top-level setting that affects all of the components within it.

Reply:
We are using a context for the read aloud component (now called "dynamic text" to allow for future merge with the glossary highlighter and possibly text translation).  I have move this code into its own provider when I do that work today.